### PR TITLE
R# for CopyTask, copy tests

### PR DESCRIPTION
**Pull request by erikma**

### CODE REVIEW

**Comment by VENESSAMARTINE03** [comment]
Please stop emailing me

File: src/Tasks.UnitTests/Copy_Tests.cs [hunk 1]
```diff
@@ -381,13 +387,25 @@ public void CopyOverReadOnlyFileParameterIsSetWithDestinationFolder()
                 Directory.CreateDirectory(destinationFolder);
 
                 using (StreamWriter sw = FileUtilities.OpenWrite(source1, true))
+                {
                     sw.Write("This is a source file1.");
+                }
+
                 using (StreamWriter sw = FileUtilities.OpenWrite(source2, true))
+                {
+
```
Comment (rainersigwald) [inline src/Tasks.UnitTests/Copy_Tests.cs]: nit: unnecessary blank line

File: src/Shared/UnitTests/MockEngine.cs [hunk 2]
```diff
@@ -198,14 +154,10 @@ internal string Log
 
                 _log.Clear();
             }
-            get { return _log.ToString(); }
+            get => _log.ToString();
```
Comment (jeffkl) [inline src/Shared/UnitTests/MockEngine.cs]: It seems weird that the getter is after the setter, do you mind fixing that?  Its like that in this whole file for some reason

File: src/Tasks.UnitTests/Copy_Tests.cs (hunk 1 above)
Comment (erikma) [reply src/Tasks.UnitTests/Copy_Tests.cs]: Fixed.

Comment (erikma) [reply src/Tasks.UnitTests/Copy_Tests.cs]: RAR reverted in next push

File: src/Shared/UnitTests/MockEngine.cs (hunk 2 above)
Comment (erikma) [reply src/Shared/UnitTests/MockEngine.cs]: Good catch. Reversed in whole file

**Pull request merged**